questions.json

```
@@ -1,182 +1,352 @@
+  "What profession doesn’t get enough credit or respect?",
+  "What memory of yours feels real but is most likely false?",
+  "What’s your “and then it got worse” story?",
+  "What was the most amazing physical feat you’ve managed to pull off?",
+  "What’s the most annoying thing about the social media platform you use most often?",
+  "If you were hired to show tourists what life is really like where you live, what would you show them / have them do?",
+  "What would be the most unsettling thing to keep occasionally finding around your house?",
+  "What nicknames do you have for people in your life?",
+  "What does the opposite sex do that you wish that you could do, but it’s not anatomically feasible or it’s socially frowned upon?",
+  "How much do you plan / prepare for the future?",
+  "What do you hate most and love most about your car?",
+  "What weird potato chip flavor that doesn’t exist would you like to try?",
+  "What’s the silliest thing you’ve convinced someone of?",
+  "How much do you think names affect the outcomes of people’s lives?",
+  "What product or service is way more expensive than it needs to be?",
+  "What’s the shadiest thing you’ve seen someone do?",
+  "What was the last situation where some weird stuff went down and everyone acted like it was normal, and you weren’t sure if you were crazy or everyone around you was crazy?",
+  "What did you eat so much of that now you hate it?",
+  "What are some of the dumbest lyrics you’ve heard in a song?",
+  "Where’s the line between soup and cereal?",
+  "What word do you always mispronounce?",
+  "What do you think you do better than 90% of people?",
+  "What would be the worst food to be liquefied and drunk through a straw?",
+  "What’s the weirdest thing about modern life that people just accept as normal?",
+  "How much of your body would you cybernetically enhance if you could?",
+  "If you wanted to slowly drive a roommate insane using only notes, what kind of notes would you leave around the house?",
+  "If you had a giraffe that you needed to hide, where would you hide it?",
+  "What’s the clumsiest thing you’ve done?",
+  "What songs do you only know the chorus to?",
+  "Think of a brand, now what would an honest slogan for that brand be?",
+  "What’s something common from your childhood that will seem strange to future generations?",
+  "What’s the most amazing place in nature you’ve been?",
+  "What’s quickly becoming obsolete?",
+  "Where is the most uncomfortable place you have ever slept?",
+  "What’s the most annoying animal you’ve encountered?",
+  "What’s your best example of correlation not equaling causation?",
+  "In what situations, do you wish you could throw down a smoke bomb and disappear?",
+  "When was the last time you were hopelessly lost?",
+  "What songs do you feel compelled to sing along with when you hear them, even if you don’t totally know all the words?",
+  "What product do you wish a company would make a “smart” version of?",
```

```suggestion
  "What product do you wish a company would make a “smart” version of?",
  "Is it morally indefensible to not submit your DNA to 23andMe or Ancestry, since it can be used to solve crimes?",
```

I want everyone's opinion on this one.

---

☝️